### PR TITLE
Documentation: Primera csp limitations

### PR DESCRIPTION
**Pull request by wdurairaj**

### CODE REVIEW

File: release-notes/v1.4.0.md [hunk 1]
```diff
@@ -27,3 +27,8 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
 
 |ID|Component |Title|Description|Workaround|
 |--|---------|-----|-----------|----------|
+|CIDG-1928|Volume Groups | Volume group deletion|Volume group can be deleted even though some PVC's are still associated to it | Due to a limitation in volume group sidecar, we are not able to prevent this operation from happening. It's better to check for the `kubectl get events` to understand the volume group deletion failures|
+|CIDG-1304|Volume Mutator| Unset description | Currently we can set description field for the PVC, but if already set, we can't unset the same| None |
+|CIDG-1305|Volume Mutator| Error validation in multiple edits | If multipled fields of PVC are edited, and if one of them is incorrect, partial edits are not done, and it results in a failure. `kubectl get events` again can be used for detecting such failure| Retry the operation of edit with all proper values by looking at the kubectl events logs|
```
Comment (suneeth51) [inline release-notes/v1.4.0.md:32]: nit: `If multiple fields of PVC are edited`

File: release-notes/v1.4.0.md [hunk 2]
```diff
@@ -27,3 +27,8 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
...
+||Volume Mutator| Incorrect PVC edit| If there is a validation failure due to incorrect PVC edit, then the operation does'nt stop until the user corrects the edit operation. This is a limitation for now.|
```
Comment (suneeth51) [inline release-notes/v1.4.0.md:33]: nit: `then the operation does not stop`

File: release-notes/v1.4.0.md [hunk 3]
```diff
@@ -27,3 +27,8 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
...
+||Replication| System limitation|Handling system limtation of RCG| If more than 60 volumes are added into a RCG (Remote Copy Group) as part of parallel volume addition, due to the stop/start of RCG internally by CSP, the backend array can reach system limits and can cause PVC creation failure. | Limit the number of PVC's to be added to a RCG when done parallely|
```
Comment (suneeth51) [inline release-notes/v1.4.0.md:34]: nit: `limitation of RCG`

nit: `Limit the number of PVCs to be added ...`

**Review by suneeth51 — CHANGES_REQUESTED** [review]

**Comment by wdurairaj** [comment]
@suneeth51 , hi , request you to review the recent changes.

Thanks.

File: release-notes/v1.4.0.md [hunk 4]
```diff
@@ -29,6 +29,6 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
...
+|CIDG-1305|Volume Mutator| Error validation in multiple edits | If multiple fields of PVC are edited, and if one of them is incorrect, partial edits are not done, and it results in a failure. `kubectl get events` again can be used for detecting such failure| Retry the operation of edit with all proper values by looking at the kubectl events logs|
+||Volume Mutator| Incorrect PVC edit| If there is a validation failure due to incorrect PVC edit, then the operation does not stop until the user corrects the edit operation. This is a limitation for now.|
+||Replication| System limitation|Handling system limtation of RCG| If more than 60 volumes are added into a RCG (Remote Copy Group) as part of parallel volume addition, due to the stop/start of RCG internally by CSP, the backend array can reach system limits and can cause PVC creation failure. Limit the number of PVCs to be added |
```
Comment (suneeth51) [inline release-notes/v1.4.0.md:34]: nit: typo `limitation`

**Review by suneeth51 — APPROVED** [review]
lgtm with one more minor comment.

File: release-notes/v1.4.0.md (hunk 4 above)
Comment (wdurairaj) [reply release-notes/v1.4.0.md]: Done.. Good catch :)

File: release-notes/v1.4.0.md [hunk 5]
```diff
@@ -27,3 +27,8 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
 
 |ID|Component |Title|Description|Workaround|
 |--|---------|-----|-----------|----------|
+|CIDG-1928|Volume Groups | Volume group deletion|Volume group can be deleted even though some PVC's are still associated to it | Due to a limitation in volume group sidecar, we are not able to prevent this operation from happening. It's better to check for the `kubectl get events` to understand the volume group deletion failures|
```
Comment (raunakkumar) [inline release-notes/v1.4.0.md:30]: Can we mention that the volume group CR gets deleted but the a volume group content and the storage backed volume group remains ?

File: release-notes/v1.4.0.md [hunk 6]
```diff
@@ -27,3 +27,8 @@ The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI
...
+|CIDG-1304|Volume Mutator| Unset description | Currently we can set description field for the PVC, but if already set, we can't unset the same| None |
+|CIDG-1305|Volume Mutator| Error validation in multiple edits | If multiple fields of PVC are edited, and if one of them is incorrect, partial edits are not done, and it results in a failure. `kubectl get events` again can be used for detecting such failure| Retry the operation of edit with all proper values by looking at the kubectl events logs|
+||Volume Mutator| Incorrect PVC edit| If there is a validation failure due to incorrect PVC edit, then the operation does not stop until the user corrects the edit operation. This is a limitation for now.|
```
Comment (raunakkumar) [inline release-notes/v1.4.0.md:33]: It is possible to associate a bug id for the last 2 Notes?

**Review by raunakkumar — CHANGES_REQUESTED** [review]

**Comment by raunakkumar** [comment]
@wdurairaj Any update on this PR?

**Review by raunakkumar — APPROVED** [review]
lgtm

**Pull request merged**